### PR TITLE
Docs: Disable ePub as a temporary work-around

### PR DESCRIPTION
**Pull request by benjaoming**
Work-around for #9816

Will merge into `diataxis/main` after.

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 version: 2
 
-formats: all
+# Disabled until we can ignore warnings in ePub
+# https://github.com/readthedocs/readthedocs.org/issues/9816
+formats:
+  - pdf
```
Comment (stsewd) [inline .readthedocs.yml:6]: we also have htmlzip enabled

**Comment by stsewd** [comment]
Also, any chance we can fix the warnings instead?

```
/home/docs/checkouts/readthedocs.org/user_builds/docs/checkouts/diataxis-main/docs/user/guides/git-integrations.rst:18: WARNING: Duplicate explicit target name: "webhook-integration-github".
/home/docs/checkouts/readthedocs.org/user_builds/docs/checkouts/diataxis-main/docs/user/guides/git-integrations.rst:41: WARNING: Duplicate explicit target name: "webhook-integration-bitbucket".
/home/docs/checkouts/readthedocs.org/user_builds/docs/checkouts/diataxis-main/docs/user/guides/git-integrations.rst:51: WARNING: Duplicate explicit target name: "webhook-integration-gitlab".
```

But I don't see any duplicates in that page, so maybe a bug with sphinx or the tabs extension.

File: .readthedocs.yml (hunk 1 above)
Comment (benjaoming) [reply .readthedocs.yml:6]: Thanks!

**Comment by benjaoming** [comment]
> Also, any chance we can fix the warnings instead?

#9816 is about finding a way to exclude certain warnings so the ePub build can go ahead despite of the warnings.

We could open an issue about fixing these warnings, but as I understand it from @humitos, we might have already decided in principle to replace sphinx-tabs with sphinx-design.

**Comment by benjaoming** [comment]
Replaced by https://github.com/readthedocs/readthedocs.org/pull/9866

**Pull request closed without merging**